src/libs/actions/Report.js

```diff
@@ -950,7 +950,7 @@ function updateLastReadActionID(reportID, sequenceNumber) {
     API.Report_UpdateLastRead({
         accountID: currentUserAccountID,
         reportID,
-        sequenceNumber: lastReadSequenceNumber,
+        sequenceNumber: lastReadSequenceNumber || 0,
```

Comment:
Ah, there's a fix in place (and merged) for this already: https://github.com/Expensify/Expensify.cash/pull/2721

We don't really want to set it to zero in case we overwrite existing stored sequences (although that should not happen, given that his should only be hit for new chats).

Anyways, fix in, so you can remove this :).

Reply:
awesome, thank you for that